jax/test_util.py

```diff
@@ -66,34 +66,33 @@ def is_sequence(x):
   else:
     return True
 
-def numpy_eq(x, y):
+
+def assert_numpy_eq(x, y):
   testing_tpu = FLAGS.jax_test_dut and FLAGS.jax_test_dut.startswith("tpu")
   testing_x32 = not FLAGS.jax_enable_x64
   if testing_tpu or testing_x32:
-    return onp.allclose(x, y, 1e-3, 1e-3, equal_nan=testing_tpu)
+    onp.testing.assert_allclose(x, y, 1e-3, 1e-3)
   else:
-    return onp.allclose(x, y)
+    onp.testing.assert_allclose(x, y)
 
 
-def numpy_close(a, b, atol=ATOL, rtol=RTOL, equal_nan=False):
+def assert_numpy_close(a, b, atol=ATOL, rtol=RTOL):
```

Comment:
I cannot think of any case when we would not want `equal_nan=True`. If both arrays are NaN, then that they should definitely match as "close."